resources/qml/delegates/TextMessage.qml

```diff
@@ -2,8 +2,10 @@ import ".."
 
 MatrixText {
 	property string formatted: model.data.formattedBody
+	property double emojiCount: timelineSettings.enlarge_emoji_only_msg ? model.data.emojiCount : 0
```

Comment:
This seems unnecessary

Reply:
I use emojiCount twice: "emojiCount > 0 && emojiCount < 4". That was used to not have to do calculations twice and only to do it if the option is enabled